packages/reactive_ble_mobile/ios/Classes/Plugin/PluginController.swift

```diff
@@ -287,20 +282,6 @@ final class PluginController {
                 $0.characteristicUuids = (service.characteristics ?? []).map { characteristic in
                     Uuid.with { $0.data = characteristic.uuid.data }
                 }
-                $0.characteristics = (service.characteristics ?? []).map { characteristic in
-                    DiscoveredCharacteristic.with{
```

Comment:
Didn't you just remove `Expose characteristic properties` from #392 ?

Reply:
I think there was a sync problem with my local fork as it has just recently been added, i'll look into it tomorrow.